docs/upgrade-guides/0.2.0.md

```diff
@@ -0,0 +1,106 @@
+# Updating js-bigchaindb-driver from v0.1.x to v0.2.0
+
+The latest version of js-bigchaindb-driver contains breaking changes to its
+external API. In this document, we enumerate all changes to allow you to make
+upgrades efficiently.
+
+Note that upgrading the js-bigchaindb-driver to v0.2.0 was done to enable
+functionality included in the latest (v1.0) BigchainDB release.  A full list of
+BigchainDB v1.0's breaking changes can be found
+[here](https://github.com/bigchaindb/bigchaindb/blob/17913dca682ff105540c0ea73365f1763efc2083/docs/upgrade-guides/v0.10--%3Ev1.0.md).
```

Comment:
Even if it's included in the link, I think it's useful to highlight also that there has been changes to the data model as well.